README.md

```diff
@@ -40,6 +40,9 @@ How to build your own jQuery
 ----------------------------
 
 First, [clone the jQuery git repo](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
+```bash 
+git clone https://github.com/jquery/jquery.git
+```
```

Comment:
We've had a concrete instruction in the past but we've deleted that on purpose to not maintain that piece of the documentation considering how many ways there are to clone the repo. The link above should suffice.